Makefile

```diff
@@ -58,7 +59,7 @@ lint: $(GOLANGCI_LINT) ## Lint the code
 .PHONY: build_code
 build_code: validate_go lint
 	@go mod vendor
-	go build "${CMD_DIR}${FL2M_BIN_FILE}"
+	VERSION=$$(date); go build -ldflags "-X 'main.Version=$$VERSION'" "${CMD_DIR}${FL2M_BIN_FILE}"
```

Comment:
Adding the data as the `Version` variable in the main package ... we use that to report version when we start flowlogs 2metrics

Reply:
Adding a version number to flowlogs2 metrics is a-bit beyond the title of this PR but I really needed that because I changed the code all the time, and the version (in this case the compile time ) is very useful to know what image you are using old or now, etc,.